src/complex/Parameters/FileSystemPathParameter.cpp

```diff
@@ -9,6 +9,17 @@
 #include <nlohmann/json.hpp>
 
 #include <stdexcept>
+#ifdef _MSC_VER
```

Comment:
```suggestion
#ifdef _WIN32
```

Reply:
What about clang on windows? i'm not sure how that behaves. I would assume it behaves like unix/linux which is why I went for MSC_VER instead of Win32, but I may be wrong on that one.

Reply:
From what I've read, I think `_WIN32` works the best here. There's some overlap between the macros but since we're using Windows specific headers I think that one is the most relevant.